frontend/apps/w3bstreaming/.env.example

```
@@ -0,0 +1,5 @@
+REACT_APP_SIGNALING_SERVER=ws://127.0.0.1:3001
+REACT_APP_STREAMING_PROGRAM_ADDRESS=0x3eb6f1f66560ee83603c8349c7959912266e7acded8b537cdbe4412937cad62a
+REACT_APP_STREAMING_NODE=wss://testnet.vara-network.io
+REACT_APP_STREAMING_IPFS_GATEWAY=
+REACT_APP_STREAMING_IPFS_NODE=
```

```suggestion
REACT_APP_CONTRACT_ADDRESS=0x3eb6f1f66560ee83603c8349c7959912266e7acded8b537cdbe4412937cad62a
REACT_APP_NODE_ADDRESS=wss://testnet.vara-network.io
REACT_APP_IPFS_GATEWAY_ADDRESS=
REACT_APP_IPFS_ADDRESS=
```
please stick to default naming in order for logPublicEnvs and errorTracking utils to work properly without any additional arguments.